src/probnum/quad/_bayesquad.py

```diff
@@ -90,9 +91,9 @@ def bayesquad(
 
     Returns
     -------
-    integral :
+    integral
```

Comment:
I removed the colons here since for multiple return objects, this renders properly actually without the colon. But I can also add the colons for consistency. Just thought that it's neat to have the correct documentation for the `bayesquad` function at least as this is the user interface. What do you think?

Reply:
I see your point, but I think that the overall effort we need to put in this will be less if we just do it the right way now.
Personally, I'm willing to accept broken docs for the time being if this results in reduced workload.
Does that make sense? Or should we handle this differently?

Reply:
no, makes sense too, I'll add the colons back.